### PR TITLE
fix(DB/lang): Totem Item Description

### PR DESCRIPTION
**Pull request by LukasVolgger**
Fix text in item_template and add translation in locales.

Select all totem: `SELECT entry, description FROM item_template WHERE description LIKE '%Counts as an Air%';` returns an amolunt of 72 entries where the description is set. 

Aligned `item_template_locale` to `item_template`. Translations for all locales except deDE taken from #2482.

## CHANGES PROPOSED:
-  Fix some missing spaces in `item_template`
-  Add translations for deDE, esES, esMX & frFR

## ISSUES ADDRESSED:
- Closes 

## TESTS PERFORMED:
- Imported SQL script on Win 10 pro x64 with HeidiSQL 11
- Checked result in DB

## HOW TO TEST THE CHANGES:

-  Import SQL script and check a few entries in DB

## KNOWN ISSUES AND TODO LIST:

- [ ]
- [ ] 

## Target branch(es):
- [x] Master

 
## How to test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

### CODE REVIEW

**Comment by Nefertumm** [comment]
Spanish looks good. Maybe @BarbzYHOOL can confirm for FR

File: data/sql/updates/pending_db_world/rev_1584982393046787000.sql [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+-- frFR locale
+UPDATE `item_template_locale` SET `Description` = 'Il compte comme un totem d''air, de terre, de feu et d''eau.' WHERE `ID` IN (22345, 22395, 22396, 23005, 23199, 23200, 24413, 25645, 27544, 27815, 27947, 27984, 28066, 28248, 28357, 28523, 29389, 30023, 31031, 32330, 33078, 33505, 33506, 33507, 33843, 33939, 33940, 33941, 33951, 33952, 33953, 35104, 35105, 35106, 37575, 38361, 38367, 38368, 39728, 40267, 40322, 40708, 40709, 40710, 42593, 42594, 42595, 42596, 42597, 42598, 42599, 42601, 42602, 42603, 42604, 42606, 42607, 42608, 42609, 45114, 45169, 45255, 46978, 47665, 47666, 47667, 50458, 50463, 50464, 51501, 51507, 51513) AND `locale` = 'frFR';
```
Comment (BarbzYHOOL) [inline data/sql/updates/pending_db_world/rev_1584982393046787000.sql]:
```suggestion
UPDATE `item_template_locale` SET `Description` = 'Compte comme un totem d''air, de terre, de feu et d''eau.' WHERE `ID` IN (22345, 22395, 22396, 23005, 23199, 23200, 24413, 25645, 27544, 27815, 27947, 27984, 28066, 28248, 28357, 28523, 29389, 30023, 31031, 32330, 33078, 33505, 33506, 33507, 33843, 33939, 33940, 33941, 33951, 33952, 33953, 35104, 35105, 35106, 37575, 38361, 38367, 38368, 39728, 40267, 40322, 40708, 40709, 40710, 42593, 42594, 42595, 42596, 42597, 42598, 42599, 42601, 42602, 42603, 42604, 42606, 42607, 42608, 42609, 45114, 45169, 45255, 46978, 47665, 47666, 47667, 50458, 50463, 50464, 51501, 51507, 51513) AND `locale` = 'frFR';
```

seems more french to me but i don't know where it is supposed to show ingame

**Comment by BarbzYHOOL** [comment]
where do they show up ingame?

**Comment by LukasVolgger** [comment]
Its the item description for the given locales like that

**Comment by Remberance** [comment]
UPDATE `item_template_locale` SET `Description` = '可以作为空气图腾、大地图腾、火焰图腾和水之图腾使用。' WHERE `ID` IN (22345, 22395, 22396, 23005, 23199, 23200, 24413, 25645, 27544, 27815, 27947, 27984, 28066, 28248, 28357, 28523, 29389, 30023, 31031, 32330, 33078, 33505, 33506, 33507, 33843, 33939, 33940, 33941, 33951, 33952, 33953, 35104, 35105, 35106, 37575, 38361, 38367, 38368, 39728, 40267, 40322, 40708, 40709, 40710, 42593, 42594, 42595, 42596, 42597, 42598, 42599, 42601, 42602, 42603, 42604, 42606, 42607, 42608, 42609, 45114, 45169, 45255, 46978, 47665, 47666, 47667, 50458, 50463, 50464, 51501, 51507, 51513) AND `locale` = 'zhCN';

this is simplified Chinese item's description.

**Comment by LukasVolgger** [comment]
Thx @Remberance for your contribution!

**Comment by FrancescoBorzi** [comment]
image

updated to better display

**Pull request merged**